tfgrid3/umbrel/dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    UMBREL_VERSION="release" \
+    UMBREL_REPO="getumbrel/umbrel" \
+    UMBREL_INSTALL_PATH="/umbrel" 
+
+RUN apt update && \
+    apt -y install wget curl vim net-tools iputils-ping openssh-server docker.io \
+    fswatch jq rsync sudo iproute2 git gettext-base python3 gnupg avahi-daemon avahi-discover libnss-mdns nginx ufw
+
+RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
```

Comment:
Docker-compose v2 is now available, stable, and should be compatible with docker-compose. the only difference is that it no longer calls `docker-compose` but `docker compose`. if no specific reason to use docker-compose v1, I would suggest using the latest version.

Reply:
upgraded to docker-compose v2.16.0, I can't only use the docker-compose plugin as umbrel uses `docker-compose` command widely and they are checking for its exist in the [start](https://github.com/getumbrel/umbrel/blob/d7c9f4d16317d52eef2193458935dc673790f4db/scripts/start#L23) script